src/greeklt/remove_accentuation.py

```diff
@@ -17,14 +17,19 @@ def remove_accentuation(string: str):
         "Ώ": "Ω",
     }
     dieresis = {"ι": "ϊ", "υ": "ϋ"}
+    dieresis_reverse = {"ϊ": "ι", "ϋ": "υ"}
     new_string = ""
     prev_char = 0
     for c in string:
         char = c
         if c in accents.keys():
             char = accents[c]
-        if c in dieresis.keys() and prev_char in ("ά", "ό", "έ"):
-            char = dieresis[c]
+        if modifier == 0:
+            if c in dieresis.keys() and prev_char in ("ά", "ό", "έ"):
+                char = dieresis[c]
+        if modifier == 1:  # Remove dieresis
+            if c in dieresis_reverse.keys():
+                char = dieresis_reverse[c]
```

Comment:
if modifier == 0 can become `if modifier:`which equals to `if modifier == True`,
and if modifier == 1 can become `if not modifier:` which equals to `if modifier == False`.